homeassistant/components/jewish_calendar/__init__.py

```diff
@@ -157,3 +170,24 @@ async def async_unload_entry(hass: HomeAssistant, config_entry: ConfigEntry) ->
         hass.data[DOMAIN].pop(config_entry.entry_id)
 
     return unload_ok
+
+
+@callback
+def async_update_unique_ids(
+    ent_reg: er.EntityRegistry, new_prefix: str, old_prefix: str
+) -> None:
+    """Update unique ID to be unrelated to user defined options.
+
+    Introduced with release 2024.6
+    """
+    for platform, descriptions in PLATFORM_DESCRIPTIONS.items():
+        for description in descriptions:
+            new_unique_id = f"{new_prefix}-{description.key}"
+            if ent_reg.async_get_entity_id(platform, DOMAIN, new_unique_id):
```

Comment:
Remove this as we're now guarding In `setup_entry` to run or not.
After first time it will have a config entry and then it should run any more as all entities should then have the new unique id

Reply:
Done